package-lock.json

```diff
@@ -1803,9 +1803,9 @@
       "dev": true
     },
     "array-differ": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-differ/-/array-differ-1.0.0.tgz",
-      "integrity": "sha1-7/UuN1gknTO+QCuLuOVkuytdQDE=",
+      "version": "3.0.0",
+      "resolved": "https://registry.npmjs.org/array-differ/-/array-differ-3.0.0.tgz",
+      "integrity": "sha512-THtfYS6KtME/yIAhKjZ2ul7XI96lQGHRputJQHO80LAWQnuGP4iCIN8vdMRboGbIEYBwU33q8Tch1os2+X0kMg==",
```

Comment:
@zaki-binary not sure why this happens, but the last time Salar update it, it case a problem, could u please check and see why it update the package-lock?

Reply:
I updated it on purpose via `npm audit fix`, as I wanted to remove the vulnerable dependencies in package-lock.